renderer/src/main.tsx

```diff
@@ -1,9 +1,24 @@
+import * as Sentry from '@sentry/react'
+import { BrowserTracing } from '@sentry/tracing'
 import React from 'react'
 import ReactDOM from 'react-dom/client'
 import { BrowserRouter } from 'react-router-dom'
 import App from './App'
 import './index.css'
 
+Sentry.init({
+  dsn: 'https://ff9615d8516545158e186d863a06a0f1@o1408530.ingest.sentry.io/6762462',
+  integrations: [new BrowserTracing()],
+
+  release: window.electron.stationMetadata.version,
+  environment: window.electron.stationMetadata.environment,
+
+  // Set tracesSampleRate to 1.0 to capture 100%
+  // of transactions for performance monitoring.
+  // We recommend adjusting this value in production
+  tracesSampleRate: 1.0
```

Comment:
submitting traces is expensive (and potentially intrusive), because a lot of data will be captured and submitted. Shall we reduce this right away, to say 10%?

Reply:
Good point. Do we actually need tracing at all? Maybe we can start with collecting errors only and wait with adding tracing until we need it?

Reply:
I'm concerned that we'll never start using traces if we don't at least have _some_ in our dashboard 🤔 But I don't feel strongly about this either, we could just remove them too

Reply:
Makes sense. I'll set the rate to `0.1` then. We can easily tweak this later, based on what we learn from real-world usage.